### PR TITLE
final corrections to inline docs

### PR DESCRIPTION
**Pull request by abhijithch**
Fresh PR  which links #182.

### CODE REVIEW

File: src/queue.jl [hunk 1]
```diff
@@ -4,6 +4,11 @@ type Queue{T}
     store::Deque{T}
 end
 
+"""
+    Queue(T[, blksize::Integer=1024])
+
+Create a `Queue` objet containing elements of type `T`.
```
Comment (DanielArndt) [inline src/queue.jl]: **object**

**Comment by DanielArndt** [comment]
LGTM other than small typo.

**Comment by abhijithch** [comment]
I might need some help with squashing.

**Comment by rawls238** [comment]
you should be able to run `git rebase -i origin/master` (or `upstream/master` depending on what you named the remote branch). and squash the commits

**Comment by DanielArndt** [comment]
@abhijithch No problem! The easiest way, if you have all your commits in a row like this is to use:

`git rebase -i origin/master`

Note, I'm assuming `origin` is the name of the JuliaLang remote, if not, you will need to modify that accordingly. When you do that, you should see a list of your commits (anything that isn't on master), and you can change `pick` to `squash` for all but the first one. That will make everything squash up into the first commit. Let me know if you have issues with that.

`git rebase origin/master` without the `-i` is usually the preferred way to "merge" changes from upstream into your working branch, to avoid all the merge commits.

**Comment by abhijithch** [comment]
@DanielArndt thanks for the `-i` tip. 
When I do a `git rebase origin/newinlinedoc`, it says `Current branch newinlinedoc is up to date.`. But my `git log` shows multiple commits.

**Comment by rawls238** [comment]
You want to be rebasing off the master branch, not your branch since your branch will always be up to date with your branch.

**Comment by abhijithch** [comment]
I think it is OK now. Had to do a force push though.

**Comment by rawls238** [comment]
lgtm, thanks!

**Comment by DanielArndt** [comment]
@abhijithch Sorry, I should have mentioned the force push. That is required any time you re-write history (rebasing, and squashing both 're-write' the history by making it look like things happened differently).

A `rebase` basically says "take my branch, and pretend I started it here". git will take anything that is different between the branches (in this case, your commits) and tack them on the end. In this way you can avoid "merge" commits (merging master into your feature branch), because everything is re-committed on top of the new state of master. You will, however, need to resolve the conflicts as they are recommitted.

Hope that makes sense. Git is usually best learned in small steps, so hopefully you learned a little :)

**Pull request merged**